api/src/main/java/uk/gov/hmcts/payment/api/controllers/PaymentController.java

```diff
@@ -423,4 +427,39 @@ public String notFound(PaymentNotFoundException ex) {
     public String return400(PaymentException ex) {
         return ex.getMessage();
     }
+
+
+    private PaymentDto filterFeeCode(PaymentDto paymentDto) {
+        LOG.info("Start Function filterFeeCode :" , paymentDto.toString());
```

Comment:
Please override the toString() method for PaymentDTO wherein the necessary fields can be logged.

Reply:
Done